scipy/special/basic.py

```diff
@@ -754,20 +754,97 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse error function.
+    This function compute the inverse error function.
+    Inverse error function is satisfying :math:`erf(erf^{-1}(x))=x`.
```

Comment:
```
The inverse error function satisfies :math:`erf(erf^{-1}(x))=x`.
```

Reply:
do we need `:math:` for this? `erf(erfinv(x)) = x` is enough in my opinion

Reply:
`erf` should be `\mathrm{erf}` within :math: to get it to typeset properly, so it's probably indeed better not write it.

Reply:
The main purpose of using `:math:` was showing a mathematical formula in description.
but I agree with @ilayn and @pv. I will modify `:math:` to erf(erfinv(x)) = x.